tests/eve-payload-07-http-gap/suricata.yaml

```diff
@@ -7,6 +7,9 @@ outputs:
       filetype: regular #regular|syslog|unix_dgram|unix_stream|redis
       filename: eve.json
       types:
+        - frame:
+            # this should not affect detection (but it did)
```

Comment:
This really shouldn't. This is output config section..
Q: Is the fix really then https://github.com/OISF/suricata/pull/11717/commits/ec69ad0a7a6d1918839b00c75314af68aa013765 ?
It seems to me that stream frame should be enabled at all times. wdyt?

Reply:
> This really shouldn't. 

Indeed, it should not affect detection but it does now in master...
The fix is indeed https://github.com/OISF/suricata/commit/ec69ad0a7a6d1918839b00c75314af68aa013765

> It seems to me that stream frame should be enabled at all times. wdyt?

Suricata is smart enough that frames are truly created only if they get used in rules or output...
So, no rules with frame:http2.stream, and no output, means these frames do not get created...